.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
please remove that file of the PR

Reply:
Could you remove that file ?

Reply:
this file is here to avoid inserting my IDE's files in the project